Code-Sleep-Python/py_api/cli_github.py

```diff
@@ -8,5 +8,7 @@
 r=rq.get("https://api.github.com/users/"+user+"/received_events",auth=(user,pwd),params=params)
 data.extend(r.json())
 print("\n")
-for i in range(len(data)):
+length = le(data)
+# this way the system wont have to compute length using len() function everytime. it saves on time complexity
```

Comment:
I dont understand this change though?
what is le?

Reply:
type error it is:
len(data) it should be.

Reply:
it really doesnt make sense? sorry have to close this PR